docs/source/upgrade-guide/index.md

```diff
@@ -30,6 +30,82 @@ the latest changes, and propose to you to merge the changes, so you can run it o
 
 ## Upgrading to Volto 16.x.x
 
+### `volto-slate` is now core
+
+From Volto 16.0.0-alpha.15 onwards, `volto-slate` is integrated into Volto core and enabled by default. This means that the default text block is now called `slate` and the former/deprecated text block `text` based on `draftJS` is now restricted (hidden) and enforced on Volto bootstrap. This is a major change and should be planned in advance before you install any version of the alphas after 16.0.0-alpha.15.
+
+````{note}
+Volto 16 is intended to be the LTS version used when Plone 6 Final is released. Volto 16 will ensure be compatible with all existing Volto deployments and won't introduce any hard breaking change. So, you'll have the choice to continue using `draftJS` text block in your projects. See below for more information.
+````
+
+```{important}
+From Volto 16 on, `draftJS` is deprecated and might be removed from core in next major versions.
+```
+
+These are the possible scenarios:
+
+- New projects
+- Existing projects, already using `volto-slate` addon
+- Existing projects using core `draftJS`, opting to continue using `draftJS`
+- Existing projects using core `draftJS`, opting to start using `slate` without migrating (possible, but not recommended)
+- Existing projects using core `draftJS`, opting to migrate to `slate`
+
+### New projects
+
+New projects are unaffected, since they will use slate as default text block from the beginning.
+
+### Existing projects, already using `volto-slate` addon
+
+For those projects using already `volto-slate` they should follow these steps in your project configuration:
+
+- Remove `volto-slate` add-on from your project build dependencies
+- Add these lines to the jest config in your project `package.json`:
```

Comment:
```suggestion
- Add these lines to the Jest configuration in your project's `package.json`:
```

Reply:
Does the user need to substitute `<rootDir>` with their rootDir value? If so, then let's explicitly state that:

```
- Add these lines to the Jest configuration in your project's `package.json`, substituting the project's root directory for `<rootDir>`:
```

Reply:
It's fine like this, it does a var interpolation internally.